packages/medusa/oas/store-spec3-base.yaml

```diff
@@ -212,29 +212,29 @@ tags:
   - name: Auth
     description: Auth endpoints that allow authorization of customers and manages their
       sessions.
-  - name: Cart
+  - name: Carts
```

Comment:
note: for reference
https://github.com/medusajs/medusa/blob/2b8c33f08b8761014312ac3bfab379296b8ad538/packages/medusa-js/src/index.ts#L53-L71